src/Serilog/Enrichers/EnvironmentUserNameEnricher.cs

```diff
@@ -1,18 +1,18 @@
 ﻿// Copyright 2013-2015 Serilog Contributors
-// 
+//
 // Licensed under the Apache License, Version 2.0 (the "License");
 // you may not use this file except in compliance with the License.
 // You may obtain a copy of the License at
-// 
+//
 //     http://www.apache.org/licenses/LICENSE-2.0
-// 
+//
 // Unless required by applicable law or agreed to in writing, software
 // distributed under the License is distributed on an "AS IS" BASIS,
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#if !PROFILE259
+#if !DOTNET5_1
```

Comment:
I couldn't come up with a good "feature" name for this. It's basically some `Environment` methods/properties that are missing in `dotnet5.1`...